### DISCUSSION TITLE
Color of exported glTF light in Bevy is different from Blender

### DISCUSSION THREAD

**Discussion by vadimb892** [bevyengine/bevy#12760] [Q&A]
I exported a glTF scene file from Blender. And when I run scene in Bevy, the light color is different. When exporting to glTF, I choose Unitless light mode. I tried to change in Color Management View Transform from Filmic to Standard. All this didn't help. Now I'm trying to fix this flaw on the Bevy side by manually changing the color of each light source. I only care about the color of the light in Bevy. What settings in Blender need to change?

Blender:
![Blender color light](https://github.com/bevyengine/bevy/assets/71188139/6f66d77f-8264-425b-ac5c-05be5a29e5e2)

Bevy:
![Bevy color light](https://github.com/bevyengine/bevy/assets/71188139/9c889c47-1bce-4c96-b578-0bf13bc58c1a)

**Comment by alice-i-cecile** [comment]
Have you ensured that the tonemapping used in Blender and Bevy match? That will have a large effect on the final perceived color, and our defaults aren't the same.
